gallery/src/pages/components/ha-alert.ts

```diff
@@ -99,7 +99,7 @@ const alerts: {
     type: "warning",
     iconSlot: html`<span slot="icon" class="image"
       ><img
-        alt=""
+        alt="Home Assistant logo"
```

Comment:
I agree with this change. The original idea here (which I failed to document) is that the logo is purely decorative, thus [should have an empty alt](https://www.w3.org/WAI/tutorials/images/decorative/). However, I can see an argument that the logo's presence suggests a "system" alert vs. some other component (integration, etc). Does the above match your reasoning for this change? Curious to hear your perspective.

A second question: Will this get localized? Does it need to be manually piped through i18n?

Reply:
Honestly, it could go either way, but this change doesn't really matter.  It's just for the "gallery" at https://design.home-assistant.io, which is not translated.  In the actual code, it's not used this way.